lib/app/components/atoms/custom_elevated_button.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+
+class CustomElevatedButton extends StatelessWidget {
+  const CustomElevatedButton({
+    super.key,
+    required this.buttonText,
+    required this.initialActiveFlag,
+    required this.onPressed,
+  });
+
+  final String buttonText;
+  final bool initialActiveFlag;
+  final void Function()? onPressed;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
+      height: 60,
+      width: 280,
+      // decoration: BoxDecoration(border: ),
+      child: ElevatedButton(
+        style: ButtonStyle(
+          shape: MaterialStateProperty.all<RoundedRectangleBorder>(
+            RoundedRectangleBorder(
+              side: initialActiveFlag
+                  ? BorderSide.none //活性
+                  : const BorderSide(
+                      color: Colors.blue,
+                      width: 2.0,
+                    ), //非活性
+              borderRadius: BorderRadius.circular(300),
+            ),
+          ),
+          foregroundColor: initialActiveFlag
+              ? MaterialStateProperty.all<Color>(Colors.white) //活性
+              : MaterialStateProperty.all<Color>(Colors.blue),
+          backgroundColor: MaterialStateProperty.all(
+            initialActiveFlag
+                ? Colors.blue //活性
+                : Colors.white, //非活性
+          ),
+        ),
+        //FIXME: isMemberSelectedがtrueの場合に、選択されたメンバーの情報を受け取って次画面へ遷移する処理を追加
+        onPressed: onPressed,
```

Comment:
```suggestion
        onPressed: onPressed?.call(),
```
callメソッドつけないと実行できない気がします。動作確認もお願いしますね。

Reply:
@king-kazu39 
ボタンを活性にした状態でCustomElevatedButtonのonTapにprintを仕込んで動作を見てみたのですが機能しているようでした。ちなみにどういった理由でcall関数をつけるべきと判断されましたか？記事をいくつか見てみたのですが使い所がしっくり来ず....

↓動作の動画
<ソース変更点>
・create_chat_group_view.dartのCustomElevatedButtonのonTapにprint('test')を実装
・create_chat_group_controller.dartのisMemberSelectedの初期値をfalseからtrueへ変更。（現状メンバー選択時にtrueにする処理が未実装のため）


https://user-images.githubusercontent.com/49933865/200780444-d6d4ecba-ac0b-48cb-b981-5cdc42155c76.mov

Reply:
>callメソッドつけないと実行できない気がします。

すいません。実行できないというのは私の勘違いでした🙇w
私もあまりちゃんと理解してなかったんですが、
`onPressed()`と`onPressed.call()`という書き方は同じらしいです。
https://zenn.dev/chooyan/articles/d46a47c2709069

あまり理解せず使っていた感じなんですが、nullチェックを簡単にするために使うといいよーって感じですかね。
あとは引数として渡ってきた関数をわかりやすくするくらいかな。この辺は好みかもw
特に気にしなければ修正しないでいいですよ。

Reply:
@king-kazu39 
ElevatedButtonの特性上onPressedで何もしない(nullを渡す)ならText widgetなど別のwidgetを使うような気がするので、今回はcall()の実装を見送りたいと思います。call()の必要性が生じたら実装します！

call()なんて普段使ってなかったので勉強になりました、ありがとうございます！

もしこれ以外で問題無さそうでしたらapproveをお願いします。